packages/tests/web-test-runner.config.js

```diff
@@ -56,16 +56,30 @@ function makeSauceLauncherOnce() {
   if (!sauceLauncher) {
     const user = (process.env.SAUCE_USERNAME || '').trim();
     const key = (process.env.SAUCE_ACCESS_KEY || '').trim();
-    if (!user || !key) {
-      throw new Error(
-        'To test on Sauce, set the SAUCE_USERNAME' +
-          ' and SAUCE_ACCESS_KEY environment variables.'
-      );
+    const tunnelIdentifier = (process.env.TUNNEL_IDENTIFIER || '').trim();
```

Comment:
Suggest `SAUCE_TUNNEL_ID` for consistency with the other variables, and for clarity.

Reply:
Sounds good